pkg/controller/linstornodeset/linstornodeset_controller.go

```diff
@@ -282,12 +256,27 @@ func (r *ReconcileLinstorNodeSet) Reconcile(request reconcile.Request) (reconcil
 	errs := r.reconcileAllNodesOnController(ctx, pns)
 
 	statusErr := r.reconcileStatus(ctx, pns, errs)
+	if statusErr != nil {
+		return reconcile.Result{}, statusErr
```

Comment:
It would make sense to have a similar warning here to the one in the controller set where we log the suppressed errors.